Language code "en" not recognized by Kokoro model

## Description

When using the Kokoro TTS model with `--lang en`, the generation fails with an `AssertionError`. The language code `en` is not mapped to a valid internal language code. The first example from README's quickstart does not work.

## Steps to Reproduce

```bash
mlx_audio.tts.generate --model mlx-community/Kokoro-82M-bf16 --text "Hello, world"
```

Or explicitly:
```bash
mlx_audio.tts.generate --model mlx-community/Kokoro-82M-bf16 --text "Hello, world" --lang en
```

## Expected Behavior

The command should succeed, with `en` being interpreted as American English (or prompting the user to choose between American/British English).

## Actual Behavior

```
Error loading model: ('en', {'a': 'American English', 'b': 'British English', 'e': 'es', 'f': 'fr-fr', 'h': 'hi', 'i': 'it', 'p': 'pt-br', 'j': 'Japanese', 'z': 'Mandarin Chinese'})
Traceback (most recent call last):
  ...
  File ".../mlx_audio/tts/models/kokoro/pipeline.py", line 86, in __init__
    assert lang_code in LANG_CODES, (lang_code, LANG_CODES)
AssertionError: ('en', {'a': 'American English', 'b': 'British English', ...})
```

## Root Cause Analysis

In `mlx_audio/tts/models/kokoro/pipeline.py`, the `ALIASES` dictionary maps common language codes to internal single-letter codes:

```python
ALIASES = {
    "en-us": "a",
    "en-gb": "b",
    "es": "e",
    "fr-fr": "f",
    "hi": "h",
    "it": "i",
    "pt-br": "p",
    "ja": "j",
    "zh": "z",
}
```

The alias mapping is applied in `KokoroPipeline.__init__()`:
```python
lang_code = lang_code.lower()
lang_code = ALIASES.get(lang_code, lang_code)
assert lang_code in LANG_CODES, (lang_code, LANG_CODES)
```

However, the plain `"en"` code is missing from `ALIASES`, so it passes through unchanged and fails the assertion.

## Proposed Fix

Add `"en"` as an alias for American English in `pipeline.py`:

```python
ALIASES = {
    "en": "a",      # Add this line
    "en-us": "a",
    "en-gb": "b",
    "es": "e",
    "fr-fr": "f",
    "hi": "h",
    "it": "i",
    "pt-br": "p",
    "ja": "j",
    "zh": "z",
}
```

Alternatively, consider also adding other common ISO 639-1 codes that users might expect to work:
- `"fr"` → `"f"` (French)
- `"pt"` → `"p"` (Portuguese)

## Workaround

Until this is fixed, use one of the following valid language codes:

| Use this | Instead of | Language |
|----------|------------|----------|
| `a` or `en-us` | `en` | American English |
| `b` or `en-gb` | - | British English |
| `e` or `es` | - | Spanish |
| `f` or `fr-fr` | `fr` | French |
| `j` or `ja` | - | Japanese |
| `z` or `zh` | - | Mandarin Chinese |

Example:
```bash
mlx_audio.tts.generate --model mlx-community/Kokoro-82M-bf16 --text "Hello, world" --lang a
```

## Environment

- mlx-audio version: 0.2.10 (installed via pip)
- Python: 3.13
- OS: macOS

--- a/mlx_audio/tts/models/kokoro/pipeline.py
+++ b/mlx_audio/tts/models/kokoro/pipeline.py
@@ -13,13 +13,16 @@ from misaki import en, espeak
 from .voice import load_voice_tensor
 
 ALIASES = {
+    "en": "a",
     "en-us": "a",
     "en-gb": "b",
     "es": "e",
     "fr-fr": "f",
+    "fr": "f",
     "hi": "h",
     "it": "i",
     "pt-br": "p",
+    "pt": "p",
     "ja": "j",
     "zh": "z",
 }

